build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
 	implementation 'org.jsoup:jsoup:1.21.1'
 
 	// Testing Frameworks
+	testImplementation 'org.jsoup:jsoup:1.21.1'
```

Comment:
You shouldn't need this here as it's already an implementation above

Reply:
oops